source/data-api.txt

```diff
@@ -12,32 +12,53 @@ Atlas Data API
    Custom HTTPS Endpoints </data-api/custom-endpoints>
    Authenticate Data API Requests </data-api/authenticate>
    Data API Examples </data-api/examples>
+   Data Formats </data-api/data-formats>
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
    :depth: 2
-   :class: singlecol
 
-The Atlas Data API is a secure, serverless HTTPS API that lets you work with
-data stored in a linked data source or run custom application logic. You
-don't need to install any database drivers or opinionated libraries to
-work with the Data API. Instead, you send requests like this from an
-HTTPS client:
+The Data API is a managed service that lets you securely work with
+data stored in Atlas using standard HTTPS requests. You can use the Data
+API to connect to MongoDB Atlas from any platform that supports HTTPS,
+including:
 
-.. code-block:: bash
+- Web browsers
+- Servers
+- CI/CD pipelines
+- Serverless & Edge compute environments
+- Mobile applications
+- Internet-Of-Things devices
+
+The Data API is not a direct connection to your database. Instead, the
+API is a fully-managed middleware service that sits between your cluster
+and the clients that send requests.
+
+You don't need to install any database drivers or opinionated libraries
+to work with the Data API. Instead, you send standard requests like the
+following:
+
+.. io-code-block::
    
-   curl -X POST \
-        https://data.mongodb-api.com/app/myapp-abcde/endpoint/hello \
+   .. input::
+      :language: bash
+
+      curl -X POST \
+        https://data.mongodb-api.com/app/myapp-abcde/endpoint/data/v1/action/insertOne \
         -H 'apiKey: <API Key>' \
         -H 'Content-Type: application/json' \
-        -d '{ "message": "Hello, world!" }'
+        -d '{
+          "dataSource": "mongodb-atlas",
+          "database": "learn-data-api",
+          "collection": "hello",
+          "document": {
+            "text": "Hello, World!",
+          }
+        }'
 
-The Data API is not a direct connection to your database. Instead, the
-API is a fully-managed middleware service that sits between your cluster
-and client apps. For each incoming request, the Data API authenticates
-the calling user, authorizes the request, runs the endpoint logic, and
-then returns an HTTPS response to the caller.
+   .. output::
+      :language: json
+
+      { "insertedId": "63dc56ac74ddb86ed3eb8474" }
 
 Endpoints
 ---------
```

Comment:
i don't think it's smthn to address in this PR, but i find it kinda funny that we're calling both generated endpoints and custom endpoints 'the data api', but then also the generated endpoints are called 'the data api'.

probably this is smthn that extends to product/product marketing. but worth bringing up imo.

curious your thoughts.

Reply:
Yeah it's silly IMO but that's what we're calling them for now. I have some ideas on how we could be clearer that I've brought up with Sumedha but I'll reintroduce them with PM again.